src/settings/components/RoleDetails/AssignUsers.js

```diff
@@ -65,7 +65,7 @@ const AssignUsers = ({ selectedUsers, roleId, refetch }) => {
         }
       }
     })();
-  }, [isSuccess, roleDetails, isAssignUsers]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [isAssignUsers]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I realize the original `eslint-disable` was not part of this PR, but doubling-down on it without explaining it remains a problem. This rule is pretty smart and gets things right most of the time. If you think you're smarter than lint, please add a comment explaining the circumstances.

Reply:
If I add all values (11 in total) it causes an infinite loop. But that probably points to a flaw in this component. It's a little awkward since I am using the callback from the FindUser Plugin to trigger `useEffect()`. May be better to move more of this work into the callback itself. I'll try that and update if successful.

Reply:
Was impractical because of how `useChunkedCQLFetch` is called, so instead I made a comment to justify `eslint-disable`.